.github/workflows/ci_cron.yml

```
@@ -11,11 +11,10 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
     with:
       envs: |
-        - macos: test-xdist
+        - macos: test-py39
           python-version: 3.9
-        - macos: test-xdist
+        - macos: test-py310
           python-version: 3.10
-        - macos: test-xdist
+        - macos: test-py311
           python-version: 3.11
-        - linux: test-rad-xdist
-        - linux: test-devdeps-xdist
+        - linux: test-py3x-devdeps
```

```suggestion
        - linux: test-devdeps
```
Adding the names simply distinguishes the jobs from one another in the github interface.